wiki/en/Running-a-Server.md

```diff
@@ -181,28 +168,31 @@ This sets the path to where server's recording will be stored. With this path se
 
 **Note**: When your server is recording, clients will display a message that recording is on.
 
-### Delay panning
+### Custom Directory Server Address
 
-This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal "volume" panning. Note that to achieve the effect, all clients should be set to 'mono in / stereo out' (or to 'stereo', but only if transmitting a stereo signal).
+Leave this field empty unless you need to list your server on a [Custom Directory](#3-directory)
 
-### Custom Directory Server Address
+### Server List Filename
+
+If you are running your server as a custom directory and need to restart it for any reason (for example when rebooting the host), servers connected to it will be disconnected until they re-register. This does not mean that clients connected to those servers will be disconnected, but does mean that new clients will be not able to see servers listed by your directory until those servers reconnect. To enable the list of registered servers to persist between restarts, specify the location and name of a file that the directory can read and write to here.  
 
-Leave this field empty unless you need to list your server on a [private Directory](#3-directory)
+### Delay panning
 
+This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal "volume" panning. Note that to achieve the effect, all clients should be set to 'mono in / stereo out' (or to 'stereo', but only if transmitting a stereo signal).
```

Comment:
> Note that to achieve the effect, all clients should be set to 'mono in / stereo out' (or to 'stereo', but only if transmitting a stereo signal).

I think that's redundant.  To hear a stereo effect, you need stereo out.  Goes without saying?

Reply:
I recall somebody was very keen to add this line when we originally did it (can't remember who) but if it's not accurate we should correct.